R/param_overview.R

```diff
@@ -0,0 +1,34 @@
+#' get an overview over all available param values in a paramval object
+#'
+#' @param pv paramval object
+#' @return named list with names corresponding to `names(pv$param)`
+#' containing the unique values for each column of `pv$param`
+#'
+#' @export
+param_overview <- function (pv) {
+  stopifnot(!is.null(pv$param))
+  unique_params <- lapply(pv$param, unique)
```

Comment:
does this work properly when `param` is a tibble?

Reply:
Yes.